sdstore/sdstore.go

```diff
@@ -32,6 +33,22 @@ type sdStore struct {
 	client *retryablehttp.Client
 }
 
+func getExpectContinueTimeout() int {
```

Comment:
[nits]
You need to comment on the function

Reply:
Thanks for the suggestion. I'll add comments to the function.

Reply:
I have fixed it.